### DISCUSSION TITLE
Badges in the README

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#640] [General]
I noticed that the FP homepage here at Github had only one badge, and it is broken, which can cause a bad impression on new visitors:

![image](https://user-images.githubusercontent.com/77707706/188524000-8be76b8b-5b95-489f-87c4-7c1dbfb8b3ad.png)

So, I played a bit with it, and included some of the impressive Freeplane numbers:

![image](https://user-images.githubusercontent.com/77707706/188524193-2882c11a-9efc-42d4-b0fe-6aac8fe4c76b.png)

What else is relevant to be included? I'm working on that [here](https://github.com/eu2021/freeplane/blob/shields-test/README.md).

**Comment by dpolivaev** [comment]
I used your badges and added links. It is live. Thanks

**Reply by euu2021** [reply]
Nice. Looks good.
